next.config.js

```diff
@@ -101,6 +101,18 @@ async function blogPostsPathMap() {
       }
     })
   })
+
+  const numPages = Math.ceil(posts.length / env.BLOG_POSTS_PER_PAGE)
+
+  for (let n = 1; n <= numPages; n += 1) {
+    pathMap[`/blog/page/${n}`] = {
+      page: Routes.RouteNames.Blog,
+      query: {
+        page: n,
+      },
```

Comment:
Do we need to worry about locale here?

Reply:
I fixed this by allowing to use language on the blog listing. However, it looks like there some kind of bug and the list is empty with english locale. It had same results with query parameter pages.